webrtc/src/main/java/com/baeldung/webrtc/SocketHandler.java

```diff
@@ -0,0 +1,32 @@
+package com.baeldung.webrtc;
+
+import org.springframework.stereotype.Component;
+import org.springframework.web.socket.TextMessage;
+import org.springframework.web.socket.WebSocketSession;
+import org.springframework.web.socket.handler.TextWebSocketHandler;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.concurrent.CopyOnWriteArrayList;
+
+@Component
+public class SocketHandler extends TextWebSocketHandler {
+
+    List<WebSocketSession> sessions = new CopyOnWriteArrayList<>();
+
+    @Override
+    public void handleTextMessage(WebSocketSession session, TextMessage message) throws InterruptedException, IOException {
+
+        for (WebSocketSession webSocketSession : sessions) {
+            if (webSocketSession.isOpen() && !session.getId()
+                .equals(webSocketSession.getId())) {
```

Comment:
This could fit on one line. If you want to keep it broken up, then line 22 would need only two spaces indentation from line 21, so that it's not the same level of indentation as the nested block (line 23).

Reply:
I have fixed this.